roseus_bt/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 2.8.3)
```

Reply:
Just confirmed it ran normally with the downgraded version as well!